api/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alpine:3.15.0
+
+RUN apk add --update --no-cache \
+  build-base \
+  nodejs-current \
```

Comment:
We have to use a specific version of nodejs (16 now) and npm (8+). 
Maybe the same would apply for some of the other dependencies?